src/server/repository/public/user/index.ts

```diff
@@ -21,3 +21,6 @@ export const UserRepository = {
   remove,
   readCountryUsersByRole,
 }
+
+export type { UsersGetManyProps }
+export { usersBuildGetManyQuery }
```

Comment:
Verbose names because these are exported on their own (unlike for example `UserRepository.getMany` where it doesn't make sense to be verbose)

Reply:
Sure. Will change it. At first I thought these `<Entity>Repository....` like `UserRepository` were reserved for actual DB operations only. 👍